README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
Hi @jkeenan,

there are three problems here:
* It's not the task of the DBIish documentation to explain the syntax for named arguments. If this is something you have stumbled over, maybe add a link to relevant documentation elswhere.
* The `:$db` example does two things at one: it mentions a new syntax *and* uses a different name for the name argument. That's a bad combination if done without any extra explanation.
* The fact that `db` happens to be an alias for `database` is likely backend specific, and should be clearly labeled as such, if we must mention it at all.

If you don't address at least the last two points, you muddle the documentation more than you clear it up.

Cheers,
Moritz